runtimes/clone/Cargo.toml

```diff
@@ -19,66 +19,67 @@ serde = {version = "1.0.137", optional = true, features = ["derive"]}
 
 # RPC
 did-rpc-runtime-api = {path = "../../rpc/did/runtime-api", default-features = false}
-frame-system-rpc-runtime-api = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.26"}
-pallet-transaction-payment-rpc-runtime-api = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.26"}
+frame-system-rpc-runtime-api = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.27"}
+pallet-transaction-payment-rpc-runtime-api = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.27"}
 
 # KILT pallets & primitives
+kilt-support = {path = "../../support", default-features = false, optional = true}
```

Comment:
Was this failing? Or why did you add it in?

Reply:
It was a clippy complaint

Reply:
Interesting. Mine does not, plus I do not see it being used anywhere inside that crate. I see that its usage has been commented out as part of the `// TODO: Enable after adding delegation pallet`, so that is probably the reason. If we don't plan to uncomment those, then perhaps we can remove it from the Cargo file as well?